### PR TITLE
fix api cursor and coordinator change

### PR DESCRIPTION
**Pull request by kvahed**

### CODE REVIEW

File: tests/simple/simple_query.go [hunk 1]
```diff
@@ -102,16 +102,7 @@ func (t *simpleTest) queryDocuments(c *collection) error {
...
-			// Request succeeded, check if coordinator is same as create-cursor request
-			if createResp.CoordinatorURL != getResp.CoordinatorURL {
-				// Coordinator changed, we expected a failure, but got a success. Not good.
-				t.queryNextBatchNewCoordinatorCounter.failed++
-				t.reportFailure(test.NewFailure("Reading next batch AQL cursor succeeded with 200, but expected a 404 because of coordinator change"))
-				t.log.Infof("Reading next batch AQL cursor succeeded with 200, but expected a 404 because of coordinator change")
-				return nil
-			}
-		}
+		}
```
Comment (neunhoef) [inline tests/simple/simple_query.go:105]: Should lines 84-89 now be removed? We only expect 404 if a coordinator we have been talking to was rebooted or vanished.

**Review by neunhoef — CHANGES_REQUESTED** [review]
See comment.

File: tests/simple/simple_query.go (hunk 1 above)
Comment (kvahed) [reply tests/simple/simple_query.go:105]: true

**Review by neunhoef — APPROVED** [review]
LGTM

**Pull request merged**